### PR TITLE
[Tyr] Add config for osm2mimir

### PR DESCRIPTION
**Pull request by azime**
Jira : [NAVP-1709](https://jira.kisio.org/browse/NAVP-1709)

### CODE REVIEW

File: source/tyr/tyr/binarisation.py [hunk 1]
```diff
@@ -160,7 +160,6 @@ def collect_metric(task_type, job, dataset_uid):
     yield
     end = datetime.datetime.utcnow()
     try:
-        logger = logging.getLogger(__name__)
```
Comment (nonifier) [inline source/tyr/tyr/binarisation.py:163]: Why removing the logger name ? did you had any trouble with this ?

File: source/tyr/tyr/binarisation.py [hunk 2]
```diff
@@ -715,31 +714,28 @@ def openaddresses2mimir(self, autocomplete_instance, filename, job_id, dataset_u
...
-            poi_types_file_name = '{}/poi-types.json'.format(os.path.dirname(working_directory))
-            with open(poi_types_file_name, 'w') as f:
-                f.write(autocomplete_instance.poi_types_json)
-            params.append('--poi-config')
-            params.append(poi_types_file_name)
-    params.append('--dataset')
-    params.append(autocomplete_instance.name)
+    filename = "custom_config"
+    custom_config_config_toml = '{}/{}.toml'.format(os.path.dirname(working_directory), filename)
+    with open(custom_config_config_toml, 'w') as f:
```
Comment (nonifier) [inline source/tyr/tyr/binarisation.py]: Shouldn't you get the configuration from the database instead ?

File: source/tyr/tyr/binarisation.py (hunk 1 above)
Comment (azime) [reply source/tyr/tyr/binarisation.py:163]: decalred in [L172](https://github.com/CanalTP/navitia/blob/5eba04b051cdc4828a622052d6de41e4c9ab44ec/source/tyr/tyr/binarisation.py#L172)

Comment (azime) [reply source/tyr/tyr/binarisation.py:163]: Unused variable

File: source/tyr/tyr/binarisation.py (hunk 2 above)
Comment (nonifier) [reply source/tyr/tyr/binarisation.py]: My bad, I did read it the wrong way around... :+1:

**Review by nonifier — APPROVED** [review]
looking good :+1:

File: source/tyr/tyr/binarisation.py [hunk 3]
```diff
@@ -715,31 +714,28 @@ def openaddresses2mimir(self, autocomplete_instance, filename, job_id, dataset_u
...
     autocomplete_instance = models.db.session.merge(autocomplete_instance)  # reatache the object
     logger = get_autocomplete_instance_logger(autocomplete_instance, task_id=job_id)
     logger.debug('running osm2mimir for {}'.format(job_id))
     job = models.Job.query.get(job_id)
     cnx_string = current_app.config['MIMIR_URL']
     working_directory = unzip_if_needed(filename)
 
-    params = ['-i', working_directory, '--connection-string', cnx_string]
-    for lvl in autocomplete_instance.admin_level:
-        params.append('--level')
```
Comment (kadhikari) [inline source/tyr/tyr/binarisation.py:727]: It means old parameters like '--level' will not be used anymore?

**Review by kadhikari — APPROVED** [review]
The unused columns like 'admin_level' and 'poi_types_json' will have to be deleted at least after one release deployed in 'prod'.

File: source/tyr/tyr/binarisation.py (hunk 3 above)
Comment (azime) [reply source/tyr/tyr/binarisation.py:727]: yes :+1:

**Pull request merged**